lib/ronin/support/network/http.rb

```diff
@@ -1068,6 +1068,31 @@ def get_cookies(path, **kwargs)
           end
         end
 
+        #
+        # Sends an HTTP request and returns the parsed `Cookie` header(s).
+        #
+        # @param [String] path
+        #   The path to to make the request for.
+        #
+        # @!macro request_kwargs
+        #
+        # @return [Array<Cookie>]
+        #   The parsed `Cookie` headers.
+        #
+        # @since 1.1.0
+        #
+        def post_cookies(path, **kwargs)
+          response = request(:post,path,**kwargs)
+
+          if (set_cookies = response.get_fields('Cookie'))
+            set_cookies.map do |cookie|
+              Cookie.parse(cookie)
+            end
+          else
+            []
```

Comment:
Let's return `nil` if there's no `Set-Cookie` header. This will make it easy to determine when no cookies were returned:

```ruby
if (cookies = http_post_cookies(url))
  ...
end
```